src/main/java/com/osiris/autoplug/client/tasks/updater/server/TaskServerUpdater.java

```diff
@@ -151,7 +151,10 @@ private void doInstallDependingOnProfile(String version, String latestVersion, S
     }
```

Comment:
Also would be great if you could enable the spigot update test again to see if this works.